index.html

```diff
@@ -17,12 +17,14 @@
               <div class="background-image-d">
                 <img src="assets/icons/header-illustration-desktop.svg" alt="header">
               </div>
-              <img class="hamburger" src="assets/icons/menu.svg" alt="Menu">
+              <div class="hamburger"></div>
+             <div class="menu-contaner">
               <ul class="header-d">
                 <li><a href="#">Portfolio</a></li>
                 <li><a href="#">About</a></li>
                 <li><a href="#">Contact</a></li>
               </ul>
```

Comment:
- Hey, let's ensure that all the links take users to the corresponding sections in your portfolio web page to meet the project requirement and to improve the user experience when clicking the links.
- Also, it is great how you've made the menu disappears when any link is clicked, however, according to the design, you should add the missing cancel button to improve your design as this would also improve the user experience. The user can decide not to visit any of the links and may decide not to click the links.  How would they go out of the mobile menu if there's no cancel button? Let's improve our design. :pray: 

*your design*
![Screenshot from 2023-04-17 17-09-39](https://user-images.githubusercontent.com/59397159/232547058-705944cc-c421-489b-911c-f69be054f99a.png)


*design template*
![Screenshot from 2023-04-17 17-08-26](https://user-images.githubusercontent.com/59397159/232547085-b4997787-3b4c-4ffa-87fc-70f2403d68b3.png)